src/components/size-chart/size-chart.hbs

```diff
@@ -9,7 +9,7 @@
     </button>
     <!-- Table START -->
     <table class="size-chart-table show-inches">
-        <caption aria-describedby="toggle-size-units-button" class="element-invisible">Size Chart</caption>
+        <caption class="element-invisible">Size Chart</caption>
```

Comment:
The content that is intended to be referred to here isn't appropriate for a describedby attribute, as it includes instruction to click to change units of measurement.